api/tests/routers/test_labels.py

```
@@ -0,0 +1,193 @@
+	)
+
+	assert response.status_code == 200
+	data = response.json()
+
+	# Verify the response data
+	assert data['dataset_id'] == test_dataset
+	assert data['user_id'] == form_data['user_id']
+	assert data['file_type'] == form_data['file_type']
+	assert data['file_alias'] == form_data['file_alias']
+	assert data['label_description'] == form_data['label_description']
+	assert data['audited'] == False
```

Ruff (0.8.0)

Avoid equality comparisons to `False`; use `if not data['audited']:` for false checks

Replace with `not data['audited']`

(E712)